### PR TITLE
Feature/functional diversity endpoints mock

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

**Comment by cazapatamar** [comment]
I won't merge this branch until Angela ends the number-species endpoints but for now, i create this PR if you want to review the functional diversity endpoints. @erikasv @Trjegul84

File: services/diversity/api_doc/functional.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/**
+ * @apiDefine s_richness Search > Species richness
+ * Information related to the species richness in a given area.
+ */
```
Comment (erikasv) [inline services/diversity/api_doc/functional.js:4]: Wrong group, this one is already defined and is missing correct description. (I see other group used from routes)

File: services/diversity/src/routes/functional.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+   * @apiName DryForestFeatures
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values of functional features in the dry forest in a given area. Features are: Leaf Area,
+   * Leaf Nitrogen, Maximun Height, Specific Leaf Area, Wood Density and Seed Mass.
+   *
+   * @apiParam (Query params) {String} areaType area type
+   * @apiParam (Query params) {String|Number} areaId area id
+   *
+   * @apiSuccess {Object} result
```
Comment (erikasv) [inline services/diversity/src/routes/functional.js:50]: result is an array of objects

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/diversity/api_doc/functional.js (hunk 1 above)
Comment (cazapatamar) [reply services/diversity/api_doc/functional.js]: Fixed

File: services/diversity/src/routes/functional.js (hunk 2 above)
Comment (cazapatamar) [reply services/diversity/src/routes/functional.js]: Fixed

**Review by erikasv — APPROVED** [review]

**Pull request merged**